client/app/bundles/momentCard/components/MomentCardDraft.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+import Tag from '../../shared/components/Tag';
+import css from './MomentCard.scss';
+
+type MomentCardDraftState = {};
+
+type MomentCardDraftProp = {
+};
+
+export default class MomentCardDraft
+  extends React.Component<MomentCardDraftProp, MomentCardDraftState> {
```

Comment:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.